cmd/kubelet/app/options/options.go

```diff
@@ -212,17 +209,11 @@ type KubeletFlags struct {
 
 // NewKubeletFlags will create a new KubeletFlags with default values
 func NewKubeletFlags() *KubeletFlags {
-	remoteRuntimeEndpoint := ""
-	if runtime.GOOS == "linux" {
-		remoteRuntimeEndpoint = "unix:///var/run/dockershim.sock"
-	} else if runtime.GOOS == "windows" {
-		remoteRuntimeEndpoint = "npipe:////./pipe/dockershim"
-	}
-
+	initOptions()
```

Comment:
Mutating global state is fragile.
It would be safer and easier to read if this either called into an os-specific helper that returned a default KubeletFlags, or just used an if statement to do the same thing.